tests/kafkatest/tests/core/reassign_partitions_test.py

```diff
@@ -129,9 +129,9 @@ def move_start_offset(self):
         time.sleep(6)
 
     @cluster(num_nodes=8)
-    @matrix(bounce_brokers=[True, False],
-            reassign_from_offset_zero=[True, False])
-    def test_reassign_partitions(self, bounce_brokers, reassign_from_offset_zero):
+    #@matrix(bounce_brokers=[True, False],
```

Comment:
Probably didn't mean to include this?

Reply:
Yeah all of this file was from debugging something else, I must've accidentally added it.